client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -502,7 +502,7 @@ export class List extends React.Component {
 						}
 					>
 						{ this.state.changePrimaryDomainModeEnabled
-							? translate( 'Abort primary domain change' )
+							? translate( 'Cancel primary domain change' )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.